apps/meteor/app/slackbridge/server/SlackAdapter.js

```diff
@@ -1007,12 +1007,11 @@ export default class SlackAdapter {
 			};
 
 			if (!isImporting) {
-				await Messages.setPinnedByIdAndUserId(
-					`slack-${slackMessage.attachments[0].channel_id}-${slackMessage.attachments[0].ts.replace(/\./g, '-')}`,
-					rocketMsgObj.u,
-					true,
-					new Date(parseInt(slackMessage.ts.split('.')[0]) * 1000),
-				);
+				const messageId = `slack-${slackMessage.attachments[0].channel_id}-${slackMessage.attachments[0].ts.replace(/\./g, '-')}`;
```

Comment:
You check for `attachments[0].text` but this both props could be still undefined 👀 

worth checking before using, or are we sure they'll be there?

Reply:
I've just added the check for both props. We're sure they'll be there, but I guess it's worth checking